pkgs/by-name/bu/buildstream/package.nix

```diff
@@ -0,0 +1,67 @@
+{
+  python3Packages,
+  fetchPypi,
+  lzip,
+  patch,
+  bubblewrap,
+  fuse3,
+  lib,
+  buildbox,
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "buildstream";
+  version = "2.4.1";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-crjsC66S4L42dejUQQ+i9j4+W68JF9Dn6kezaN3EZF8=";
+  };
```

Comment:
Any reason for not using the GitHub repo?
https://github.com/apache/buildstream

Reply:
Yes, or else he would not have made the remark, and on top of that, you'll get the tests in the archive :)